src/components/Rank/InvalidRankCardItem.tsx

```diff
@@ -57,8 +57,7 @@ export const InvalidRankCardItem = ({
               width="100%"
               justifyContent="end"
             >
-              {/* <AiFillCaretUp color={upIndicationIconColor} /> */}
-              <Text fontWeight="bold" fontSize={{ md: 'xs', base: 'xs' }}>
+              <Text fontWeight="bold" fontSize={{ base: 'xs' }}>
```

Comment:
```suggestion
              <Text fontWeight="bold" fontSize="xs" >
```
please check if it is same, if it is same we can directly use "xs".

Reply:
Alright, I’ll look into i!t.